src/postgraphql/graphiql/index.html

```diff
@@ -0,0 +1,21 @@
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>GraphiQL</title>
+    <meta name="robots" content="noindex"/>
+    <style>
+      html, body, #root {
+        height: 100%;
+        margin: 0;
+        overflow: hidden;
+        width: 100%;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root">
+    </div>
+    <script>window.GRAPHQL_PATH = '{GRAPHQL_PATH}'</script>
+    <script src="/_postgraphql/graphiql.js"></script>
```

Comment:
You might want to throw this in:

```html
<script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
```

Just in case the browser doesn’t support `fetch` :+1: